src/library/traktor/traktorfeature.cpp

```diff
@@ -414,48 +414,43 @@ void TraktorFeature::parseTrack(QXmlStreamReader &xml, QSqlQuery &query) {
 TreeItem* TraktorFeature::parsePlaylists(QXmlStreamReader &xml) {
 
     qDebug() << "Process RootFolder";
-    //Each playlist is unique and can be identified by a path in the tree structure.
+    // Each playlist is unique and can be identified by a path in the tree structure.
     QString current_path = "";
-    QMap<QString,QString> map;
+    QSet<QString> playlists;
 
-    QString delimiter = "-->";
+    const QString delimiter = "-->";
```

Comment:
this avoids runtime Utf8 to Utf16 conversion: 
```suggestion
    const QString delimiter = QStringLiteral("-->");
```